src/main/scala/cromwell/engine/backend/local/LocalBackend.scala

```diff
@@ -22,16 +32,35 @@ class LocalBackend extends Backend {
     commandWriter.write(commandLine)
     commandWriter.flushAndClose()
 
-    s"/bin/bash $commandFile" ! ProcessLogger(stdoutWriter write, stderrWriter write)
+    s"/bin/bash $commandFile" ! ProcessLogger(stdoutWriter writeWithNewline, stderrWriter writeWithNewline)
 
     Seq(stdoutWriter, stderrWriter).foreach {
       _.flushAndClose()
     }
 
     taskOutputs.map { taskOutput =>
-      taskOutput.name -> taskOutput.expression.evaluate(
+
+      val rawValue = taskOutput.expression.evaluate(
         scopedLookupFunction(call, symbolStore),
         new LocalEngineFunctions(TaskExecutionContext(stdoutFile, stderrFile)))
+
+      val possiblyAutoConvertedValue = rawValue.map {
+        case v: WdlString =>
+          // Handle possible auto-conversion from string literal to WdlFile
+          // and possible magic "stdout" or "stderr" literals.
+          taskOutput.wdlType match {
```

Comment:
Since it looks like this whole thing might change w/ the recent discussion of stdout/stdin I'm going to not worry too much about this structure for now